recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
     - python
     - async-timeout >=3.0,<4.0
     - attrs >=17.3.0
-    - chardet >=2.0,<4.0
+    - chardet >=2.0,<5.0
```

Comment:
This is pinned at `<4` in the current release: https://github.com/aio-libs/aiohttp/blob/2f655a59d0daedfa2a794996c4355b576c98ecc8/setup.py#L69

Reply:
Yes, I saw that. It is in fact why the tests are failing. They have merged to master a commit back in December that updates to allow chardet 4.0 (https://github.com/aio-libs/aiohttp/pull/5333), but there have been no releases since then. We would like to support environments that include chardet 4. Can we cherry-pick the patch #5333 since it only allows the new chardet (no other related code changes)?

Reply:
I added the change in PR5333 to a patch, and now everything passes. Please review. Thanks!